src/tup/version.h

```diff
@@ -22,6 +22,6 @@
 #define tup_version_h
 
 /* This constant is defined by an automatically generated file */
-extern const char *tup_version;
```

Comment:
I believe you could make the value constant by using 

`extern const char *const tup_version;`